modules/pipeline/dbclient/op_pipeline_report.go

```diff
@@ -120,7 +120,7 @@ func (client *Client) DeletePipelineReportsByPipelineID(pipelineID uint64, ops .
 	session := client.NewSession(ops...)
 	defer session.Close()
 
-	_, err := session.ID(pipelineID).Delete(&spec.PipelineReport{})
```

Comment:
why `.ID` didn't work?

Reply:
if use `ID` interface, will generate sql: `DELETE FROM `dice_pipeline_reports` WHERE `id`=? []interface {}{0x9896f2}`, pipelineid is not the primary key of pipeline_reports table

and pipelineID is not `string` type